services/auth/src/api.js

```diff
@@ -1060,6 +1060,9 @@ builder.declare({
     req.clientId(),
     req.scopes(),
   ]);
+  //setting headers as purposed by bug 161375
+  req.params.test_auth = {clientId, scopes};
+  res.setHeader('X-test_auth', {clientId, scopes});
```

Comment:
I apologize for contracting jhford, but I don't think a header is a good choice here -- all of the other data about the request comes back in the response body.  Also, the Taskcluster client libraries don't make response headers available, so there would be no way to look for this header.  So please change this bit to include `queryString` in the response (`res.reply`) instead.